examples/tutorial.ipynb

```diff
@@ -297,20 +318,53 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "You can also get the sets of image files for each duplicated set using `imagelab.info`"
+    "You can also get exactly which images are there in each (exact/near) duplicated set using `imagelab.info`. The code below shows how to do that."
```

Comment:
```suggestion
    "You can also get the sets of image files for each (exact/near) duplicated set using `imagelab.info`"
```

Reply:
the original text seems much shorter, and not any less clear to me?

Reply:
I wanted to replace files to be more generic. can rewrite to 
`You can also get exactly which images are there in each (exact/near) duplicated set using `imagelab.info`.`